### PR TITLE
Events for monthly upgrade clicks.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: src/components/monthly-upgrade.js [hunk 1]
```diff
@@ -41,6 +46,12 @@ var MonthlyUpgrade = React.createClass({
       return;
     }
 
+    reactGA.event({
```
Comment (alanmoo) [inline src/components/monthly-upgrade.js]: This fires after the submission goes through completely?

Comment (alanmoo) [reply src/components/monthly-upgrade.js]: That's based on the check for `state.submitting`...I'm trying to think through if there's any opportunities for this to give us bad data.

Comment (ScottDowne) [reply src/components/monthly-upgrade.js]: No, on the button click.

Comment (ScottDowne) [reply src/components/monthly-upgrade.js]: I think, let me double check. On click was my intent.

Comment (ScottDowne) [reply src/components/monthly-upgrade.js]: Yeah, it's onClick

**Pull request merged**